src/index.ts

```diff
@@ -1,2 +1,3 @@
-export * from './theme';
```

Comment:
Decide to keep the chakra and extend thems for the storybook but not expose them to the consumer, we can also drop them without a breaking change in the future this way

Reply:
@GutierrezGo Regarding your comment. I changed the export so we can only use the tokens we define outside the storybook.
So far I found one case in the email templates where we currently use the components https://github.com/smg-automotive/email-templates/pull/22#discussion_r894533581